bgpd/bgp_zebra.c

```diff
@@ -882,6 +882,12 @@ bool bgp_zebra_nexthop_set(union sockunion *local, union sockunion *remote,
 			 */
 			if (!v6_ll_avail && if_is_loopback(ifp))
 				v6_ll_avail = true;
+			else {
+				flog_warn(
```

Comment:
Not `zlog_info`? Isn't `ferr_bgp_err` designed for errors?

Reply:
This is an error in configuration.  As such it should be an error or warning that the user needs to fix